### PR TITLE
[DB-979] [risk=no] Data Disclaimers

### PR DESCRIPTION
**Pull request by mhigbyflowers**

### CODE REVIEW

File: public-ui/src/app/data-browser/chart/chart.component.ts [hunk 1]
```diff
@@ -610,11 +610,10 @@ export class ChartComponent implements OnChanges, AfterViewInit {
         legendText = seriesName + ', Medical Concept';
         ageResult = this.domainCountAnalysis.ageCountAnalysis.results.
           filter(x => x.stratum4 === a.stratum2)[0];
-        totalCount = (ageResult <= 20) ? '&le; 20' : ageResult.countValue;
+        totalCount = (ageResult.countValue <= 20) ? '&le; 20' : ageResult.countValue;
```
Comment (mhigbyflowers) [inline public-ui/src/app/data-browser/chart/chart.component.ts:613]: fixed an issue I found in the chart component

File: public-ui/src/app/shared/components/pop-up/pop-up.component.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  @Output() closed: EventEmitter<any> = new EventEmitter;
+  showStatement = true;
+  body: string;
+
+
+  close() {
+    this.showStatement = false;
+    this.closed.emit();
+  }
+}
```
Comment (mhigbyflowers) [inline public-ui/src/app/shared/components/pop-up/pop-up.component.ts]: new popup component

File: public-ui/src/app/shared/components/pop-up/pop-up.component.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+@Component({
+  selector: 'app-pop-up',
+  templateUrl: './pop-up.component.html',
+  styleUrls: ['./pop-up.component.css', '../../../styles/template.css']
+})
+export class PopUpComponent {
+  @Input() statement: string;
+  @Input() title: string;
+  @Output() closed: EventEmitter<any> = new EventEmitter;
+  showStatement = true;
```
Comment (dolbeew) [inline public-ui/src/app/shared/components/pop-up/pop-up.component.ts]: This variable seems redundant. The `showStatement` vars in the parents that get updated by the `closed` emitter should already handle the showing and hiding of the statement.

File: public-ui/src/app/shared/components/pop-up/pop-up.component.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  body: string;
```
Comment (dolbeew) [inline public-ui/src/app/shared/components/pop-up/pop-up.component.ts]: Doesn't look like this is being used anywhere.

File: public-ui/src/app/shared/components/pop-up/pop-up.component.ts (hunk 3 above)
Comment (mhigbyflowers) [reply public-ui/src/app/shared/components/pop-up/pop-up.component.ts]: thanks for catching these. Forgot to remove them when refactoring.

**Review by dolbeew — APPROVED** [review]
LGTM

**Pull request merged**